app/models.py

```diff
@@ -342,6 +354,7 @@ class Session(BaseModel):
     user_id: str
     quiz_id: str
     created_at: datetime = Field(default_factory=datetime.utcnow)
+    events: Optional[List[Event]]
```

Comment:
after reading a bit more on `Optional`, i think it should be `events: List[Event] = None` .. we don't intend it to be optional param, we just iniitialize with `None` to mean ki its a default value. eventually it will get populated.

Reply:
Makes sense
So I guess this depends on the situation
Can you review the `models.py` file and see if any situation has an extra `Optional` which should not be there?